packages/maskbook/src/plugins/ITO/UI/CompositionDialog.tsx

```diff
@@ -1,66 +1,85 @@
 import { useCallback, useState } from 'react'
-import { createStyles, DialogContent, DialogProps, makeStyles, Typography } from '@material-ui/core'
+import { DialogContent, DialogProps } from '@material-ui/core'
 import { InjectedDialog } from '../../../components/shared/InjectedDialog'
+import { getActivatedUI } from '../../../social-network/ui'
+import { useChainId } from '../../../web3/hooks/useChainState'
 import { ITO_MetaKey } from '../constants'
 import type { ITO_JSONPayload } from '../types'
-import { useStylesExtends } from '../../../components/custom-ui-helper'
+import type { AbstractTabProps } from '../../../extension/options-page/DashboardComponents/AbstractTab'
+import AbstractTab from '../../../extension/options-page/DashboardComponents/AbstractTab'
 import { useI18N } from '../../../utils/i18n-next-ui'
-import AbstractTab, { AbstractTabProps } from '../../../extension/options-page/DashboardComponents/AbstractTab'
-import { editActivatedPostMetadata } from '../../../social-network/ui'
-import { TestForm } from './TestForm'
+import { CreateItoGuide } from './CreateItoGuide'
+import React from 'react'
```

Comment:
remove it.

Reply:
We're using React 17 new JSX transform so you don't need to import React

Reply:
it removed